src/Microsoft.Azure.SignalR.AspNet/DispatcherHelper.cs

```diff
@@ -53,6 +53,7 @@ internal static ServiceHubDispatcher PrepareAndGetDispatcher(IAppBuilder builder
 
             // Get the one from DI or new a default one
             var router = configuration.Resolver.Resolve<IEndpointRouter>() ?? new DefaultEndpointRouter();
+            var clientResultsManger = configuration.Resolver.Resolve<IClientResultsManager>();
 
```

Comment:
Do we need to resolve? `AspNet` SignalR doesn't support client invocation, shall we set null in below dependent component?

Reply:
not needed. fixed in #1684